repo_tests/frontend_service_only_path_guard_test.py

```
@@ -0,0 +1,82 @@
+live tree. ``frontend_service_only_path_guard_contrast_test.py`` supplies a
+fixture that SHOULD trip the detector and one that must not; this file
+supplies the live half.
+
+The floor below is bound to the sweep's REACH -- the number of frontend calls
+discovered -- never to the count of findings. A floor bound to findings passes
+in silence the moment the scanner starts returning nothing.
+"""
+
+from __future__ import annotations
+
+from typing import Iterable
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)